DashWallet/Sources/UI/Explore Dash/DWExploreTestnetViewController.h

```diff
@@ -32,6 +33,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface DWExploreTestnetViewController : UIViewController <DWNavigationFullscreenable>
 
 @property (nullable, nonatomic, weak) id<DWExploreTestnetViewControllerDelegate> delegate;
+@property (nonatomic, strong) id<DWSyncProtocol> syncModel;
```

Comment:
We need access to the sync state in the `DWExploreTestnetViewController`, and passing a model around view controllers seems to be the accepted approach. Maybe there is another way though.

Reply:
@Syn-McJ I think it's okay for now, but for a long term we need to refactor this part and instead of passing one instance of syncModel between all viewcontrollers (we will need this in coinbase later and some other places), we better have some global property of the sync state (or some shared SyncManager) that you can access from any view controller or view model.

Reply:
Sounds good